{"name": "test_login_with_invalid_credentials", "status": "passed", "steps": [{"name": "Вход в систему с недействительными учетными данными", "status": "passed", "steps": [{"name": "Регистрация ответа", "status": "passed", "attachments": [{"name": "Текст ответа", "source": "8e02f606-d3c7-45a4-91de-b71aef8b5ba2-attachment.json", "type": "application/json"}, {"name": "Заголовки", "source": "ecce9014-eeb0-4402-902f-e2122644ccb1-attachment.json", "type": "application/json"}], "start": 1719734254973, "stop": 1719734254974}], "start": 1719734253858, "stop": 1719734254974}], "start": 1719734253858, "stop": 1719734254974, "uuid": "480e45a2-ab36-4ad6-84a9-893887f14823", "historyId": "15d5d744ddd2cc4bbf96e98f1487c81f", "testCaseId": "15d5d744ddd2cc4bbf96e98f1487c81f", "fullName": "tests.test_user_operations.TestUserOperations#test_login_with_invalid_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('user_page', 'user_data', 'created_user')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_operations"}, {"name": "subSuite", "value": "TestUserOperations"}, {"name": "host", "value": "sanya"}, {"name": "thread", "value": "17172-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_operations"}]}